react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
+ReactDOM.render(
+  <Accordion />,
```

Comment:
Your accordion should receive its topics via props here. You can declare your variable in index.jsx

Reply:
Hi Tim, these changes were made. Did I do them correctly?